_posts/2022-05-31-scala-on-app-service.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Scala on App Service"
+toc: true
+toc_sticky: true
+author_name: Denver Brittain
+excerpt: "Run a Scala App using Java SE on App Service"
+category: java
+---
+
+## Prerequisites
+
+This sample is based off of the [Play Framework Hello World Tutorial](https://github.com/playframework/play-samples/tree/2.8.x/play-scala-hello-world-tutorial). 
+
+To follow the steps in this tutorial, you will need the correct version of Java, sbt, and (optionally) Maven or the Azure CLI.
+
+* Java 11
+* sbt 1.3.4 or higher. Note: if you downloaded this project as a zip file from <https://developer.lightbend.com>, the file includes an sbt distribution for your convenience.
+* Maven (Optional)

```

Comment:
If you are using Maven commands, I think you need to remove the Optional label for Maven. Same on line 14.